asmcomp/i386/proc.ml

```diff
@@ -107,7 +107,7 @@ let word_addressed = false
    These globals are denoted by Incoming and Outgoing stack locations
    with negative offsets, starting at -64.
    Unlike arguments passed on stack, arguments passed in globals
-   do not prevent tail-call elimination.  The caller stores arguments
+   do not prevent tailcall elimination.  The caller stores arguments
```

Comment:
It looks like you want to go beyond fixing obvious typos and into copy editing territory.  Beware of dragons, esp. if (since?) you are not a native English speaker.

To answer some of your questions:
* "parentheses" is the plural of "parenthesis", the latter being singular
* 'analyse/analyze' - 'specialised/specialized' - 'finalise/finalize' - 'signaled/signalled' -  'labeled/labelled' are all correct spellings. Mind you, a word can have several correct spellings in English just like it can in French (think 'clé/clef' and 'je m'assois/je m'assieds').  Some of those spellings are more common in the UK and others are more common in the US.  Depending on which native Englsh speaker you ask, you'll get different answers as to which spelling is "right".

So, is this the battle you want to fight?